src/components/Footer/LogosRow.jsx

```diff
@@ -19,7 +19,11 @@ const LogosRow = function (props) {
         </div>
         <div className="ds-l-col ds-l-col--9 ds-l-sm-col--10 ds-l-md-col--auto">
           <p className="ds-u-font-size--small ds-u-color--muted ds-u-measure--base ds-u-margin--0">
-            {props.t('footer.address')}
+            {props.t('footer.address1')}
+            <span className="registered-symbol">
+              <sup>&#174;</sup>
+            </span>{' '}
```

Comment:
I'm not sure what the implementation would end up being exactly, but I think it would be something like:
```json
"address": "A federal government website managed and paid for by the U.S. Centers for Medicare & Medicaid Services. 7500 Security Boulevard, Baltimore, MD 21244. Health Insurance Marketplace<0 /> is a registered trademark of the Department of Health and Human Services"
```
```js
<Trans
  i18nKey="footer.address"
  t={props.t}
  components={[(
    <span className="registered-symbol">
      <sup>&#174;</sup>
    </span>
  )])
/>
```

Reply:
let's give it a try